pulsar-client-cpp/lib/ProducerImpl.cc

```diff
@@ -500,9 +506,26 @@ void ProducerImpl::sendAsyncWithStatsUpdate(const Message& msg, const SendCallba
                 return;
             }
 
-            sendMessage(OpSendMsg{msgMetadata, encryptedPayload,
-                                  (chunkId == totalChunks - 1) ? callback : nullptr, producerId_, sequenceId,
-                                  conf_.getSendTimeout(), 1, uncompressedSize});
+            OpSendMsg op =
+                OpSendMsg{msgMetadata, encryptedPayload, (chunkId == totalChunks - 1) ? callback : nullptr,
+                          producerId_, sequenceId,       conf_.getSendTimeout(),
+                          1,           uncompressedSize};
```

Comment:
```c++
            OpSendMsg op{msgMetadata, encryptedPayload, (chunkId == totalChunks - 1) ? callback : nullptr,
                         producerId_, sequenceId,       conf_.getSendTimeout(),
                         1,           uncompressedSize};
```

Simplify the code. BTW, it can also reduce an invocation of the copy constructor, though the copy might be optimized due to the compiler.